README.md

```diff
@@ -0,0 +1,164 @@
+# *Alternate Bit Protocol Simulator*
+
+## FILES ORGANIZATION: Carleton University
+---
+### *Authors:*  
+---
+### Originally Developed by	: Dr. Cristina Ruiz Martin
```

Comment:
Improvements that can be done :
1) You can provide a table of contents with a hyperlink
2) You can specify how to install cadmium and destimes either giving the steps or by referring the documents
3) You can also provide hyperlinks to the author so that one can access their profile directly